docs/src/content/docs/pt-br/resources/showcase.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Vitrine Starlight
+description: Descubra sites construídos com Starlight!
+sidebar:
+  label: Vitrine Starlight
```

Comment:
Noting that in the English the sidebar label and title are not the same: the page title is “Starlight Showcase” but the sidebar label is “Site Showcase”. I guess the same might apply here?

Reply:
Great catch, thank you for pointing that out, @delucis. I've just updated the translation with a new commit.